.github/workflows/pytest.yml

```
@@ -6,6 +6,12 @@ on:
   schedule:
     # Every Monday at 7AM UTC
     - cron: '0 07 * * 1'
+  workflow_dispatch:
+    inputs:
+      dummy:
+        description: 'Place holder'
```

Could you please give this a more descriptive name?
```suggestion
        description: 'Manual dispatch'
```

---

Whoop's, I copied and pasted from another workflow, we could just remove the extra things and have `workflow_dispatch:` without the `inputs`